.github/workflows/aws-deploy.yml

```diff
@@ -55,5 +61,5 @@ jobs:
 #        uses: youyo/aws-cdk-github-actions@v2
 #        with:
 #          cdk_subcommand: 'deploy'
-#          cdk_args: '--require-approval never --context env=dev --progress events --debug'
+#          cdk_args: '--require-approval never --context env=<context> --progress events --debug'
```

Comment:
Just a placeholder to be changed when uncommenting?

Reply:
yes, that is correct.